dev/sbt-plugin/src/sbt-test/sbt-plugin/distribution/build.sbt

```diff
@@ -22,8 +22,8 @@ lazy val playProj = (project in file("playProj")).enablePlugins(PlayJava, LagomP
 val checkStartScriptLagomProj = taskKey[Unit]("checkStartScriptLagomProj")
 val checkStartScriptPlayProj = taskKey[Unit]("checkStartScriptPlayProj")
 
-checkStartScriptLagomProj := checkStartScriptTask(lagomProj)
-checkStartScriptPlayProj := checkStartScriptTask(playProj)
+checkStartScriptLagomProj := checkStartScriptTask(lagomProj).value
```

Comment:
`value` is necessary for sbt 1.2.x. Maybe 0.13 was doing some implicit conversion?

Reply:
Not it wasn't. Either it was broken in 0.13 or you were using `<<=`, which doesn't require `.value`.